dbt_platform_helper/utils/validation.py

```diff
@@ -312,6 +327,10 @@ def dbt_email_address_regex(key):
             Optional("versioning"): bool,
             Optional("lifecycle_rules"): [LIFECYCLE_RULE],
             Optional("data_migration"): DATA_MIGRATION,
+            Optional("external_role_access"): {LOWER_ALPHANUMERIC: EXTERNAL_ROLE_ACCESS},
+            Optional("cross_environment_service_access"): {
+                LOWER_ALPHANUMERIC: CROSS_ENVIRONMENT_SERVICE_ACCESS
+            },
             Optional("external_role_access"): {EXTERNAL_ROLE_ACCESS_NAME: EXTERNAL_ROLE_ACCESS},
```

Comment:
Is this a duplicate of the `external_role_access` key above. Looks like `EXTERNAL_ROLE_ACCESS_NAME` is the same as `LOWER_ALPHANUMERIC`

Reply:
Good spot. Merge error probably. will sort out.